src/editor/Gui.cpp

```diff
@@ -2617,6 +2716,121 @@ void Gui::drawAbout() {
 	ImGui::End();
 }
 
+void Gui::drawMergeWindow() {

```

Comment:
can you leave the merging stuff out of this PR? There's a lot more work that needs to be done to get this working properly.

Reply:
I revert changes and it can be works as in command line)